vm/actor/src/builtin/paych/mod.rs

```diff
@@ -128,7 +128,7 @@ impl Actor {
         }
 
         if !sv.secret_pre_image.is_empty() {
-            let hashed_secret: &[u8] = &rt.syscalls().hash_blake2b(&params.secret)?;
+            let hashed_secret: &[u8] = &rt.syscalls().hash_blake2b(&params.secret).unwrap();
```

Comment:
let me be more specific:
```
            let hashed_secret: &[u8] = &rt
                .syscalls()
                .hash_blake2b(&params.secret)
                .map_err(|e| *e.downcast::<ActorError>().unwrap())?;
```
Downcasting the error to an actor error, if it is not that, then panic (unwrap). This is fine because we expect if there is an error, it is an ActorError from running out of gas (or future proofing for anything else)